Patch by Robert Scheck <robert@fedoraproject.org> for popt >= 1.13, which solves the
multilib/multiarch problem by removing the timestamp within the footer from the doxygen
generated files.

--- popt-1.13/Doxyfile.in				2007-05-25 19:36:23.000000000 +0200
+++ popt-1.13/Doxyfile.in.multilib			2007-12-15 17:07:34.000000000 +0100
@@ -659,7 +659,7 @@
 # each generated HTML page. If it is left blank doxygen will generate a 
 # standard footer.
 
-HTML_FOOTER            = 
+HTML_FOOTER            = footer_no_timestamp.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading 
 # style sheet that is used by each HTML page. It can be used to 
--- popt-1.13/footer_no_timestamp.html			1970-01-01 01:00:00.000000000 +0100
+++ popt-1.13/footer_no_timestamp.html.multilib		2007-12-14 22:21:35.000000000 +0100
@@ -0,0 +1,5 @@
+<hr size="1"><address style="text-align: right;"><small>Generated for $projectname by&nbsp;
+<a href="http://www.doxygen.org/index.html">
+<img src="doxygen.png" alt="doxygen" align="middle" border="0"></a> $doxygenversion </small></address>
+</body>
+</html>
